src/main/scala/ch/epfl/bluebrain/nexus/iam/realms/Realms.scala

```diff
@@ -246,7 +248,17 @@ object Realms {
     */
   def index[F[_]: Timer](implicit as: ActorSystem, rc: RealmsConfig, F: Async[F]): RealmIndex[F] = {
     implicit val cfg: KeyValueStoreConfig = rc.keyValueStore
-    KeyValueStore.distributed("realms", (_, resource) => resource.rev, mapError)
+    implicit val monadError: MonadError[F, IamError] = new MonadError[F, IamError] {
```

Comment:
why do we need this additional definition?

Reply:
because we want to retry only on IamError